src/main/java/seedu/address/model/AddressBook.java

```diff
@@ -16,7 +19,8 @@
 public class AddressBook implements ReadOnlyAddressBook {
 
     private final UniquePersonList persons;
-
+    private final UniquePersonList personBeingViewed;
```

Comment:
Why do you have to use a UniquePersonList for personBeingViewed? Cant it just be a Person?